ospo-book/chapters/02-chapter.md

```diff
@@ -1 +1,107 @@
+
+* **The tension between the need to ship product features and the need to contribute back to open source:** Open source contributions may take a back seat when dealing with multiple day-to-day tasks.
+
+* **Procurement processes with never-ending steps:** Open source is a dynamic ecosystem and contributions to it should occur as smoothly and naturally as possible. The long procurement processes faced in highly regulated environments, such as finance companies and governments, create a barrier to open source contribution and engagement.
+
+* **A lack of consciousness about organizational responsibility:** Due to the way open source was taught in the past, engineering-based tools, or even the engineering jargon used, the concept of open source may not be taken seriously in other areas of the organization involved in decision-making processes, management, or policy making.
+
+
+To fully overcome these and other problems when engaging in open source, organizations need to be equipped to manage open source operations effectively on both cultural and practical levels.
+
+To understand the value of an OSPO, the reader needs to get familiar to the reasons behind (1) starting an OSPO and (2) sustaining an OSPO in the long term
+
+![ospowork](https://user-images.githubusercontent.com/43671777/232471414-8ea7c899-a13c-47de-bafe-836830525d02.png)
+
+*[Source:OSPOs, key lever for open source sustainability](https://speakerdeck.com/anajsana/ospos-a-key-lever-for-open-source-sustainability)* 
+
+## The reasons behind STARTING an OSPO
+
+The [business value of the OSPO](https://project.linuxfoundation.org/hubfs/LF%20Research/TODO%20Group%20-%20The%20Business%20Value%20of%20the%20OSPO%20-%20Report.pdf?hsLang=en) explains some of the reasons shared by Open Source leaders across different industries and organization sizes.
+* Building standardized processes around open source
+* Learn how to approach the open source community
+* Embracing the Sustainability of Open Source Projects
+* Managing Compliance 
+* Expanding access to open knowledge
+* Improving development velocity
+* Mitigating Security Risks
+
+## The reasons behind SUSTAINING an OSPO
+
+Stopping the work of an OSPO could have significant negative impacts on those organizations that use open source at any level, including loss of open source expertise, increased security risks, reduced community engagement, and damage to reputation.
+
+An OSPO needs to be an ongoing initiative within an organization in order to evolve its culture and open-source knowledge, helping the organization to contribute to and build more secure open-source software, as well as improving the sustainability of open-source projects. 
```

Comment:
I would add that OSPO also helps in using open source software. In my experience this is one of bigest responsibilities of an OSPO. 

```suggestion
An OSPO needs to be an ongoing initiative within an organization in order to evolve its culture and open-source knowledge, helping the organization to use, to contribute to and build more secure open-source software, as well as improving the sustainability of open-source projects. 
```